log4j-mongodb/pom.xml

```diff
@@ -142,4 +141,100 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+
+      <id>docker</id>
+
+      <!--
+        ~ Only the `ubuntu` CI runners have access to Docker
+        -->
+      <activation>
+        <os>
+          <family>linux</family>
+        </os>
+        <property>
+          <name>env.CI</name>
+          <value>true</value>
+        </property>
+      </activation>
```

Comment:
What activation do you propose? On the presence of `/var/run/docker.sock`?

Reply:
I replaced the `CI`-check with a check for a `/var/run/docker.sock` file: https://github.com/apache/logging-log4j2/pull/2922/commits/bbd4e7df620331c297049035cd70488ce17d5aeb

Reply:
@ppkarwasz, I did not have an objection to the previous activation condition, as a matter of fact, I support it. I just wanted to double-check if this was a conscious decision, since it won't run by default on the machine of the person-who-I-dont-want-to-call-his-name. :sweat: Reverted back to the original activation in  c8f156c67e2cd4ec28ee8b7992a1515418bc38c0.